.github/workflows/generate_dmsrc.yml

```diff
@@ -0,0 +1,45 @@
+# Generates rust_g.dm as a GitHub Actions Artifact
+name: Generate DMSrc
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v2
+        - uses: actions/setup-python@v2
+        # Generate the actual .dm file.
+        - uses: jannekem/run-python-script-action@v1
```

Comment:
You shouldn't generate it like this.

`cargo build` already creates the .dm file through the build script.